test/e2e/protocol/proposer.test.mjs

```diff
@@ -244,94 +249,38 @@ describe('Basic Proposer tests', () => {
-
-    // Note that the test operates with one proposer, which is why we use the current proposer
-    // to call `changeCurrentProposer`, in reality this proposer would be the least interested
-    let error = null;
-    try {
-      await bootProposer.changeCurrentProposer();
-    } catch (err) {
-      error = err;
-    }
-    expect(error.message).to.satisfy(message =>
-      message.includes('Transaction has been reverted by the EVM'),
-    );
-  });
-
   it('Should be able to make a block any time as soon as there are txs in the mempool', async () => {
     // User balance in L2 before making deposit
-    const balancesBeforeBlockProposed = await user.checkNightfallBalances();
+    const balancesBeforeBlockProposed = await getBalance();
 
     // Make deposit, then make block to settle the deposit
     const value = String(transferValue * 2);
-    await user.makeDeposit({
-      tokenContractAddress: String(erc20Address),
-      value,
-    });
-    await bootProposer.makeBlockNow();
+    await nf3User.deposit(erc20Address, tokenType, value, tokenId, fee);
+    await nf3Proposer.makeBlockNow();
 
     // Wait before checking user balance in L2 again
     await web3Client.waitForEvent(eventLogs, ['blockProposed']);
-    const balancesAfterBlockProposed = await user.checkNightfallBalances();
+    const balancesAfterBlockProposed = await getBalance();
 
     // Assertions
-    expect(balancesBeforeBlockProposed).to.be.an('object').that.is.empty;
-    expect(balancesAfterBlockProposed).to.have.property(erc20Address);
-
-    const erc20balances = balancesAfterBlockProposed[erc20Address];
-    expect(erc20balances).to.have.lengthOf(1);
-    expect(String(erc20balances[0].balance)).to.have.string(value); // Balance is in Wei, so we compare strings
-  });
-
-  it.skip('Should change the current proposer', async function () {
-    // SKIP We need to spin a second optimist if we want to keep this test
-    const CHANGE_PROPOSER_NO_TIMES = 8;
-    let numChanges = 0;
-    for (let i = 0; i < CHANGE_PROPOSER_NO_TIMES; i++) {
-      try {
-        const currentSprint = await getCurrentSprint();
-        const currentProposer = await getCurrentProposer();
-        console.log(
-          `     [ Current sprint: ${currentSprint}, Current proposer: ${currentProposer.thisAddress} ]`,
-        );
-
-        console.log('     Waiting blocks to rotate current proposer...');
-        const initBlock = await web3.eth.getBlockNumber();
-        let currentBlock = initBlock;
-
-        while (currentBlock - initBlock < rotateProposerBlocks) {
-          await new Promise(resolve => setTimeout(resolve, 10000));
-          currentBlock = await web3.eth.getBlockNumber();
-        }
-
-        // const res = await secondProposer.changeCurrentProposer();
-        // expectTransaction(res);
-        numChanges++;
-      } catch (err) {
-        console.log(err);
-      }
-    }
-    expect(numChanges).to.be.equal(CHANGE_PROPOSER_NO_TIMES);
```

Comment:
Removed skipped test

Reply:
don't we need a change current proposer test?

Reply:
We do. But for that we need to launch an environment with multiple optimists and this test won't do that. I believe there's some kind of ping-pong where proposer "rotation" is being tested.

Reply:
okay, thanks